### ISSUE TITLE
nvda very slow after doing a search in the cisco page, link in the called

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#7813]
### Steps to reproduce:
Open the page:
https://www.cisco.com/c/pt_br/about/events-schedule/quintas-quinze.html

press ctrl + home to go to the top of the page.
press ctrl + nvda + f to open the nvda search
type 2017
Press enter to search the page.
Now try navigating the page with the arrows.

### Expected behavior:
you can navigate the page efficiently right after you search.

### Actual behavior:
nvda is first stuck unable to navigate the page for a few seconds, then loses the focus of the result several times before finally locating the result by pressing ctrl + home and repeating the search.

### System configuration:
NVDA version:
> Example: next-14675

NVDA Installed or portable:
instaled
Other information:
> Example: Running in a VM

Windows version:
> Example: Windows 10 Version 1709 Build 16299.64

Name and version of other software in use when reproducing the issue:
firefox 57 64bits
### Other questions:

Does the issue still occur after restarting your PC?
yes
Have you tried any other versions of NVDA?
> Please list them and the result

yes 
the same result.

nvda is slower in firefox 57, however when doing the search in this site it seems that nvda is facing much more difficulty to load the browsing mode and to stay focused on the result.

**Comment by Brian1Gaff** [comment]
For what its worth and  considering my Portuguese is non existent, I tried 
this in windows 7 64 with 32 bit Firefox 52 ESR and it worked as expected, 
taking me straight to the year after the year  legend on the page in this 
issue, and no delays or issues with lag or loss of focus were seen.

So it could be the version of firefox,  the fact that its running in a 
virtual box or a windows 10 thing.

Brian

**Comment by fernando-jose-silva** [comment]
Thank you very much for testing.
Can nv acces do something right now, or do you have to wait for the mosila?

**Comment by Brian1Gaff** [comment]
I think  we are waiting for a few more people to test it. 
I was merely contributing what I see here on my particular system.
 Brian

**Comment by ehollig** [comment]
This probably has to do with Firefox 57. I am using Windows 10,1709 and can replicate this issue.
A similar experience can be found when searching for "1914" on the [World War I Wikipedia article](https://en.wikipedia.org/wiki/World_War_I).

**Comment by Brian1Gaff** [comment]
Yes I'd say that its just very s l o w .
In 52 ESR it seems very snappy to me.
 Brian

**Comment by jcsteh** [comment]
Is anyone able to test this with Firefox 58 beta or Firefox 59 nightly? There are significant performance enhancements in 58 and more still in 59 which should hopefully resolve this.

**Comment by fernando-jose-silva** [comment]
thank you so much.
I installed firefox 59, and it seemed to be a bit faster, however, when testing again on the cisco page:
The first survey I do for 2017 the nvda
arrives to give focus to the result 2017, but soon afterwards the focus returns to the beginning of the page.
By pressing nvda + f3 to do the search again the focus is posed in the result correctly.
As comparative in firefox 57 when doing the search for 2017 the first time on my machine nvda hangs for about 8 seconds and then the focus returns to the top of the page.
Pressing nvda + f3 focuses on the search result.
In firefox 57 if you press ctrl + home and do the search by 2017 the focus never stays in the result the focus always returns to the top of the page, the only way to focus on the result is soon after returning to the beginning of the page press nvda + f3, never pressing ctrl + home.
In firefox 59 one can press ctrl + home that the result gains focus, more only after the first search fails.

**Comment by fernando-jose-silva** [comment]
Steps to reproduce:
Open the page
https://www.cisco.com/c/pt_br/about/events-schedule/quintas-quinze.html
press ctrl + nvda + f
type 2018
press enter to do the search.
Expected behavior:
The nvda should take around 6 seconds to allow navigation with the arrows on the page.
Actual behavior:
nvda is taking about 50 seconds to allow navigation with the arrows
System configuration:
NVDA version:
Example: next-14799
NVDA Installed or portable:
Other information:
core i 3 6200u 2,2ghz
4gb ran
ssd 128gb
Windows version:
Example: Windows 10 Version 1709 16299.192
Name and version of other software in use when reproducing the issue:
firefox 57 64bits
Other questions:
Does the issue still occur after restarting your PC?
yes

The only changes that have occurred in my system since the report of this problem until today where the slowness has greatly increased were the software updates:
currently nvda next 14799
windows 10 build 16299.192
firefox 57.0.4.

**Comment by Adriani90** [comment]
@fernando-jose-silva I am testing this in Firefox 66.0.3 with NVDA 2019.1.1. I don't notice any performance lack. NVDA takes about one second or maybe two to respond. Do you still have this issue?

**Comment by fernando-jose-silva** [comment]
Now everything happens well.
I still find nvda when working with firefox a bit slow, but now its perfectly possible to use it.
After searching on this page within 1 second you can already navigate with the arrows.

**Comment by Adriani90** [comment]
Closing as works for me.
